CODE_OF_CONDUCT.md

```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-nico@php-perfect.de.
+dev@akiba.at.
```

Comment:
I don't think it's great to remove Nico, but instead change the link to https://github.com/ddev/ddev/blob/4e6bd6c7919b3ae12bc326481912312dc0d580e8/docs/content/users/code-of-conduct.md ?